projects/pgai/pgai/cli.py

```diff
@@ -211,8 +233,8 @@ def vectorizer_worker(
             run_vectorizer(db_url, vectorizer, concurrency)
         if once:
             return
-        time.sleep(poll_interval)
         log.info(f"sleeping for {poll_interval_str} before polling for new work")
+        time.sleep(poll_interval)
```

Comment:
![falling-down-lol-gif-by-americas-funniest-home-videos](https://github.com/user-attachments/assets/67aea862-c08e-4024-a8a2-81a203f1461e)

Reply:
When I notice it, I facepalmed hard. BTW I added that xD